en-US/mozorg/internet-health/privacy-security.lang

```diff
@@ -57,14 +67,33 @@ We should all be able to choose – with clarity and confidence – what informa
 Source: <a href="%(pew)s" rel="external">Pew Research Center</a>, May 2015
 
 
+# This is used as a decorative element in a callout box.
+;91%
+91%
+
+
 ;What you can do about user control
 What you can do about user control
 
 
+## TAG: internet_health_subpage_rebrand
```

Comment:
This tag is not available at the beginning of the file, you need to add it there two.